lib/timers.js

```diff
@@ -538,8 +548,15 @@ exports.setInterval = function(callback, repeat, arg1, arg2, arg3) {
 
 function createRepeatTimeout(callback, repeat, args) {
   repeat *= 1; // coalesce to number or NaN
-  if (!(repeat >= 1 && repeat <= TIMEOUT_MAX))
+  if (!(repeat >= 1 && repeat <= TIMEOUT_MAX)) {
```

Comment:
Is it feasible to avoid duplicating code in `createSingleTimeout()` and `createRepeatTimeout()`?

Reply:
Is the `emitWarning` call the duplicate code you're referring to?

Reply:
The warning message differs slightly.

Reply:
Yeah, that's what I was going to say. Doesn't seem worth abstracting, IMO.

Reply:
Yup, sorry, didn't notice the difference in warning messages.